docs/internal/internal-user-interface.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Internal User Interface
```

Comment:
I think internal is a bit redundant as we're already in the internal section

Reply:
Yeah, I get what you mean, but I think we should keep the title consistent with the navigation menu so that there is no confusion.